STACpopulator/implementations/CMIP6_UofT/collection_config.yml

```diff
@@ -4,4 +4,17 @@ description: Coupled Model Intercomparison Project phase 6
 keywords: ['CMIP', 'CMIP6', 'WCRP', 'Climate Change']
 license: "CC-BY-4.0"
 spatialextent: [-180, -90, 180, 90]
-temporalextent: ['1850-01-01', null]
\ No newline at end of file
+temporalextent: ['1850-01-01', '2500-01-01']
+
+assets:
+  homepage:
+      title : Project homepage
+      href : https://wcrp-cmip.org/cmip-phase-6-cmip6/
+      description: CMIP6 project homepage
+      media_type: text/html
+  license:
+      title : License
+      href : https://raw.githubusercontent.com/WCRP-CMIP/CMIP6_CVs/main/LICENSE-CC-BY
+      description: License for CMIP6 data
+      media_type: text/plain
```

Comment:
This should be added in `links` with `rel: license` instead of `assets`: 
https://github.com/radiantearth/stac-spec/blob/master/collection-spec/collection-spec.md#license

Reply:
So, I took this from your example: https://hirondelle.crim.ca/stac/collections/EuroSAT-subset-train. You have license in assets. Is that wrong?

Reply:
Yes. Looks like a misuse.

The `license` in `links` is specifically recommended by STAC. 
It is also commonly used, for example: https://planetarycomputer.microsoft.com/api/stac/v1/collections/landsat-c2-l2

Reply:
I've now updated the assets to links.